qemu/tests/qemu_img_check_fragmentation.py

```diff
@@ -0,0 +1,43 @@
+import re
+
+from avocado.utils import process
+
+from virttest import data_dir
+from virttest import storage
+
+
+def run(test, params, env):
+    """
+    Check file fragmentation.
+    1. Create a raw image with 10G.
+    2. Create a badly fragmented file with qemu-img bench.
+    3. Check file fragmentation. The extents should less than 10000.
+       With 1 MB extents, the theoretical maximum for a 10 GB image
+       is 10000 extents (10000 * 1 MB = 10 GB)
```

Comment:
we don't use a factor 1024 here?

Reply:
it's developer's comment. I will update it. thanks